DDDEastAnglia/Views/Account/ExternalLoginConfirmation.cshtml

```diff
@@ -12,6 +12,33 @@
     <fieldset>
         <legend>Associate your @ViewBag.ProviderDisplayName account</legend>
 
+        @if (string.Equals("google", ViewBag.ProviderDisplayName, StringComparison.InvariantCultureIgnoreCase))
```

Comment:
Could this be factored out into a partial view? I don't mind whether or not that partial contains the `if` statement too -whichever you think best.